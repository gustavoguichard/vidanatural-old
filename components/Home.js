import { memo } from 'react'
import Logo from 'components/Logo'
import ImageContainer from 'components/ImageContainer'
// import { Tile } from 'react-bulma-components'

import 'styles/home.scss'

export default memo(() => (
  <>
    <ImageContainer src="/static/home-bg.jpg" id="home" contentClass="home-bg">
      <Logo clickable className="home-logo" />
    </ImageContainer>
  </>
))

// <Tile>
// <Tile>
// <ImageContainer src="/static/bg.jpg" contentStyle={{ height: 300 }} />
// </Tile>
// <Tile>
// <ImageContainer src="/static/bg.jpg" />
// </Tile>
// <Tile>
// <ImageContainer src="/static/bg.jpg" />
// </Tile>
// <Tile>
// <ImageContainer src="/static/bg.jpg" />
// </Tile>
// </Tile>
