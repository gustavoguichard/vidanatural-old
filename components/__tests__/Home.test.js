import { render } from 'react-testing-library'
import Home from 'components/Home'

it('renders correctly', () => {
  const { container } = render(<Home />)
  expect(container).toMatchSnapshot()
})
