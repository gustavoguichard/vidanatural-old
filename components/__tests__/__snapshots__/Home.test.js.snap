// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="image-container"
    id="home"
  >
    <img
      class="bg-image cover"
      src="/static/home-bg.jpg"
      style="display: none;"
    />
    <div
      class="bg-image cover"
      style="background-image: url(/static/home-bg.jpg);"
    />
    <div
      class="content-above home-bg"
    >
      <div
        class="home-logo"
        style="display: block; background: url(/static/logo-white.png) no-repeat center; background-size: contain; height: 117px; width: 130px; overflow: hidden; text-indent: -999em; cursor: pointer;"
      >
        <h1>
          Vida Natural
        </h1>
      </div>
    </div>
  </div>
</div>
`;
